cli/flox/src/commands/mod.rs

```diff
@@ -375,6 +369,45 @@ impl FloxArgs {
     }
 }
 
+/// Print general welcome message with short usage instructions
+/// and give hints for creating and activating environments.
+/// List active environments if any are active.
+fn print_welcome_message(envs: EnvRegistry, active_environments: ActiveEnvironments) {
+    let welcome_message = {
+        let version = FLOX_VERSION.to_string();
+        formatdoc! {r#"
+            flox version {version}
+
+            Usage: flox OPTIONS (init|activate|search|install|...) [--help]
+
+            Use `flox --help` for full list of commands and more information
+        "#}
+    };
+
+    message::plain(welcome_message);
+
+    // print trailer message
+    // - if no environments are known to Flox yet, hint at creating one
+    // - if no environments are active, hint at activating one
+    // - if environments are active, list them
+
+    if envs.entries.is_empty() {
+        message::plain("First time? Create an environment with `flox init`\n");
+        return;
+    }
+
+    if active_environments.last_active().is_none() {
+        message::plain("No active environments. Activate one with `flox activate`\n");
```

Comment:
Good idea to add helper text. Instead of prompting to use activate (which is in the helper text after you `init`), let's use this to help discover `flox envs`. 

```
No active environments. Use `flox envs` to list all environments. 
```